ivy/functional/frontends/paddle/nn/functional/pooling.py

```diff
@@ -24,3 +25,37 @@ def avg_pool1d(
         ceil_mode=ceil_mode,
         data_format=data_format,
     )
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+def avg_pool3d(
+    x,
+    kernel_size,
+    stride=(1, 1, 1),
+    padding=(0, 0, 0),
+    /,
+    *,
+    data_format="NDHWC",
+    count_include_pad=False,
+    ceil_mode=False,
+    divisor_override=None,
```

Comment:
Reposition the arguments to mimic the exact function in the paddle paddle function [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/avg_pool3d_en.html#avg-pool3d)